common/web/keyboard-processor/src/keyboards/activeLayout.ts

```diff
@@ -141,6 +141,22 @@ namespace com.keyman.keyboards {
       return this.displayLayer + '-' + this.coreID;
     }
 
+    static sanitize(rawKey: LayoutKey) {
+      if(typeof rawKey.width == 'string') {
+        rawKey.width = parseInt(rawKey.width, 10); // '' => NaN.
+      }
+      rawKey.width = (isNaN(rawKey.width as number) ? 0 : rawKey.width) || ActiveKey.DEFAULT_KEY_WIDTH;
+
+      if(typeof rawKey.pad == 'string') {
+        rawKey.pad = parseInt(rawKey.pad, 10);
+      }
+      rawKey.pad = (isNaN(rawKey.pad as number) ? 0 : rawKey.pad) || ActiveKey.DEFAULT_PAD;
```

Comment:
I think `''` is supposed to be equivalent to `DEFAULT_PAD`, not `0`.

Reply:
Already done as-is, as elaborated upon in https://github.com/keymanapp/keyman/pull/7486#discussion_r1000054770.